packages/documentation/src/pages/getting-started/internal-tools.mdx

```diff
@@ -1,153 +1,134 @@
-| Jenkins              | http://jenkins.vetsgov-internal                  |
-| Grafana              | http://grafana.vetsgov-internal                  |
-| Sentry               | http://sentry.vetsgov-internal                   |
-| Prometheus (dev)     | http://prometheus-dev.vetsgov-internal:9090/     |
-| Prometheus (staging) | http://prometheus-staging.vetsgov-internal:9090/ |
-| Prometheus (prod)    | http://prometheus-prod.vetsgov-internal:9090/    |
-| Prometheus (utility) | http://prometheus-utility.vetsgov-internal:9090/ |
-
-You do not need to run the SOCKS proxy while you're developing unless you need access to one of the above tools.
-
-## Design Rationale
-
-Some utilities, such as metrics dashboards, error reporting, and deployment
-tools should not be made available to the public. While such a utility may lock
-functionality behind a user login system, these vary among implementations and
-may be vulnerable to exploits. Additionally, due to the difficulty of creating
-subdomains for vets.gov, it may be problematic to host some of these utilities
-when they do not support relative path installations.
-
-The preferred solution is to host these systems on an internal network that
-prevents public access. While the VA maintains an internal network, the vets.gov
-team operates within the AWS environment. Installation of a utility like this on
-the VA network would require additional authority to operate extensions,
-a server within the VA, and mapping new connections from AWS to the VA. Since
-these tools are for use exclusively by the vets.gov services installed within
-AWS, utilizing the VA's internal network is both unnecessary, more complex,
-time intensive, and would likely impact performance of the utility and
-application requiring it.
-
-To address this issue we've configured an internal DNS server which can be
-accessed via a SOCKS proxy. The SOCKS proxy is a server on your local system
-which tunnels HTTP and DNS traffic to a jumpbox on the vets.gov AWS network.
-Once connected, a developer will have access to the `*.vetsgov-internal` TLD,
-and can use their browser to connect to tools such as Sentry.
-
-Internal systems will not require any modification to connectivity, and should
-communicate with the utilities directly. They may use a `/etc/hosts` entry for
-the corresponding `*.vetsgov-internal` address when necessary.
-
-
-
-## Configuring the SOCKS proxy
-
-These steps assume you're running on Linux or OSX. There are slightly
-different commands to connect to the proxy depending on whether you
-are connected to the VA network or not. You will need to run the SOCKS
-proxy on your local system whenever you need access to tools on the
-`vetsgov-internal` domain.
-
-#### Create an SSH public key
-
-If you don't already have a SSH public key, or you're not sure if you do,
-here are the steps to create one:
-
-1. First run the following comman in your terminal: `cd ~/.ssh`.
```

Comment:
Struggled with this content a bit but decided it's best to give explicit steps. I didn't find a way to do this _in the browser_ so I migrated this to a clone-edit-push process.